lib/patientFinder/mongoid.yml

```diff
@@ -0,0 +1,34 @@
+development:
+  sessions:
+    default:
+      database: patid
```

Comment:
We are still figuring out how we want to implement multiple databases ourselves, but my understanding is that usually applications to not look across databases (so instead this would usually be implemented by creating a separate collection in the same database)